kie-server-parent/kie-server-services/kie-server-services-common/src/main/java/org/kie/server/services/impl/KieServerImpl.java

```diff
@@ -1014,8 +1017,29 @@ private void updateExtensions(KieModuleMetaData metadata, KieContainerInstanceIm
 
     public ServiceResponse<KieServerStateInfo> getServerState() {
         KieServerStateInfo state = getInternalServerState();
-        if(state != null) {
-            return new ServiceResponse<KieServerStateInfo>(ServiceResponse.ResponseType.SUCCESS, "Successfully loaded server state for server id " + KieServerEnvironment.getServerId(), state);
+        if (state != null) {
+            KieServerConfig kieServerConfig = state.getConfiguration();
+            List<KieServerConfigItem> kieConfigItems = kieServerConfig.getConfigItems();
+            ListIterator<KieServerConfigItem> listIterator = kieConfigItems.listIterator();
+            while (listIterator.hasNext()) {
+                KieServerConfigItem configItem = listIterator.next();
+                if (configItem.getName().equals(KieServerConstants.CFG_KIE_PASSWORD)) {
```

Comment:
It would be safer for avoiding NPE to compare the other way around, though it's not very likely that name is null.
```suggestion
                if (KieServerConstants.CFG_KIE_PASSWORD.equals(configItem.getName())) {
```

Reply:
@gmunozfe Thanks you for review. I made suggested changes.